common-libs/vc-common/src/verifier/util.ts

```diff
@@ -27,57 +27,61 @@ type Validations<T> = { [k in keyof T]: Validator | Validator[] }
 
 export type ValidateFn<T> = (data: Unvalidated<T>) => Promise<Validatied<T>>
 
-export const genValidateFn = <T>(validations: Validations<T>): ValidateFn<T> => async (data) => {
-  const keys = Object.keys(validations)
-  const errors: ErrorConfig[] = []
-
-  for (const _fieldName of keys) {
-    const fieldName = _fieldName as keyof T
-    const validator: Validator | Validator[] = validations[fieldName]
-    const validators: Validator[] = validator instanceof Array ? validator : [validator]
-
-    for (const validator of validators) {
-      try {
-        const outcome = await validator(data[fieldName], data)
-
-        if (outcome !== true) {
+export const genValidateFn = <T>(validations: Validations<T>): ValidateFn<T> => {
```

Comment:
How are changes in this file related to the issue?

Reply:
tsdx injected it's own eslint config which wasn't the same as affinidi's eslint config. The change away from tsdx brings this project's eslint config in line with the rest of affinidi and so this file needed a reformat.

Reply:
I see